### PR TITLE
COR-972 Apply Sanity keys logic to VR & GM pages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**

### CODE REVIEW

**Review by VWSCoronaDashboard26 — COMMENTED** [review]
Not seeing anything out of the ordinary, but what about `packages/app/src/pages/veiligheidsregio/[code]/index.tsx` and `packages/app/src/pages/veiligheidsregio/index.tsx` as files that require updating? I think these mainly apply some text for meta data (such as browser tab titles, descriptions, etc.), but I'd still like to validate if this is the case.

File: packages/app/src/pages/veiligheidsregio/[code]/sterfte.tsx [hunk 1]
```diff
@@ -37,24 +37,24 @@ import {
...
 import { ArticleParts, PagePartQueryResult } from '~/types/cms';
 import { replaceVariablesInText } from '~/utils';
-
 import { getLastInsertionDateOfPage } from '~/utils/get-last-insertion-date-of-page';
+import { useDynamicLokalizeTexts } from '~/utils/cms/use-dynamic-lokalize-texts';
 
 const pageMetrics = ['deceased_cbs', 'deceased_rivm'];
 
+const selectLokalizeTexts = (siteText: SiteText) => ({
+  caterogyTexts: siteText.common.veiligheidsregio_layout.headings.besmettingen,
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/veiligheidsregio/[code]/sterfte.tsx:46]: Small typo: `caterogyTexts` should be `categoryTexts`.

File: packages/app/src/pages/veiligheidsregio/[code]/sterfte.tsx [hunk 2]
```diff
@@ -101,7 +101,8 @@ const DeceasedRegionalPage = (props: StaticProps<typeof getStaticProps>) => {
   const { deceased_cbs: dataCbs, deceased_rivm: dataRivm, difference } = data;
 
   const { commonTexts } = useIntl();
-  const { caterogyTexts, textVr, textShared } = pageText;
+  const { caterogyTexts, textVr, textShared } =
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/veiligheidsregio/[code]/sterfte.tsx:104]: Ditto.

File: packages/app/src/pages/artikelen/[slug].tsx [hunk 3]
```diff
@@ -83,9 +85,14 @@ export const getStaticProps = createGetStaticProps(
 );
 
 const ArticleDetailPage = (props: StaticProps<typeof getStaticProps>) => {
-  const { pageText, content, lastGenerated } = props;
+  const { content, lastGenerated } = props;
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/artikelen/[slug].tsx:88]: Any particular reason why you opted to no longer destructure `pageText` from the props here?

**Review by VWSCoronaDashboard26 — COMMENTED** [review]
Theoretically approved, but a few small 'issues' and a general question.

File: packages/app/src/pages/artikelen/[slug].tsx (hunk 3 above)
Comment (VWSCoronaDashboard24) [reply packages/app/src/pages/artikelen/[slug].tsx:88]: It's good to keep it destructured indeed. I made some changes.

**Comment by VWSCoronaDashboard24** [comment]
> Not seeing anything out of the ordinary, but what about `packages/app/src/pages/veiligheidsregio/[code]/index.tsx` and `packages/app/src/pages/veiligheidsregio/index.tsx` as files that require updating? I think these mainly apply some text for meta data (such as browser tab titles, descriptions, etc.), but I'd still like to validate if this is the case.

@VWSCoronaDashboard26 These pages only consume   `commonTexts` from the `useIntl` hook which already worked before (translating the content to sanity keys) :-)

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Pull request merged**